lib/index.js

```diff
@@ -59,6 +59,7 @@ import { default as Signable } from './types/signable'
 import { default as SignedDocument } from './types/signedDocument'
 import { default as SignedString } from './types/signedString'
 import { default as SigningKey } from './types/signingKey'
+import { default as Auth } from './auth'
```

Comment:
I think the import is renamed here.  I think in auth.js and in client.js line 32, this variable keeps the same name.  I think it makes things easier to reason about if variables are not renamed.  So perhaps you could choose one of the two and make the file name and the class name and the import names all the same.  Do you have a preference to do it this way?  There could be a pattern I am not catching.

Reply:
Nope, it just used to be named Auth, but I missed this in the rename